share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -0,0 +1,128 @@
+# This file is currently being customized for quartz/toss_3_x86_64_ib.
+
+spack:
+  concretization: separately
+  view: false
+  config:
+    concretizer: clingo
+    install_tree:
+      root: /home/software/radiuss
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  packages::
+    all:
+      permissions:
+        read: world
+      providers:
+        mpi: [mvapich2]   # white238
+        #mpi: [mvapich2, openmpi]  # Is this why craylib-sci shows up?
+        # Spack defaults (since limiting the scope)
+        D: [ldc]
+        awk: [gawk]
+        blas: [openblas, amdblis]
+        daal: [intel-daal]
+        elf: [elfutils]
+        fftw-api: [fftw, amdfftw]
+        flame: [libflame, amdlibflame]
+        gl: [mesa+opengl, mesa18, opengl]
+        glx: [mesa+glx, mesa18+glx, opengl]
+        glu: [mesa-glu, openglu]
+        golang: [gcc]
+        iconv: [libiconv]
+        ipp: [intel-ipp]
+        java: [openjdk, jdk, ibm-java]
+        jpeg: [libjpeg-turbo, libjpeg]
+        lapack: [openblas, amdlibflame]
+        mariadb-client: [mariadb-c-client, mariadb]
+        mkl: [intel-mkl]
+        mpe: [mpe2]
+        #mpi: [openmpi, mpich]
+        mysql-client: [mysql, mariadb-c-client]
+        opencl: [pocl]
+        osmesa: [mesa+osmesa, mesa18+osmesa]
+        pil: [py-pillow]
+        pkgconfig: [pkgconf, pkg-config]
+        rpc: [libtirpc]
+        scalapack: [netlib-scalapack, amdscalapack]
+        sycl: [hipsycl]
+        szip: [libszip, libaec]
+        tbb: [intel-tbb]
+        unwind: [libunwind]
+        yacc: [bison, byacc]
+        uuid: [util-linux-uuid, libuuid]
```

Comment:
I'm not I understand the reason for all these providers specified here, but you know your stack better than I do, maybe they're required in order for you to get what you want.  I'm just pointing it out in case it's a holdover from running pipelines on some other infrastructure.

Reply:
I'm using double-colon so these are to add the defaults to the package.  Perhaps that's not needed in the cloud?

Reply:
They (or a subset) *are* still needed in the cloud.